azure-pipelines-ci.yml

```diff
@@ -162,11 +164,11 @@ stages:
         pip install -r requirements_test.txt -c homeassistant/package_constraints.txt
       displayName: 'Create Virtual Environment & Install Requirements'
       condition: and(succeeded(), ne(variables['CacheRestored'], 'true'))
-    - task: 1ESLighthouseEng.PipelineArtifactCaching.SaveCacheV1.SaveCache@1
-      displayName: 'Save artifacts based on Requirements'
-      inputs:
-        keyfile: 'requirements_all.txt, requirements_test.txt, .cache'
-        targetfolder: './venv'
+    - task: 1ESLighthouseEng.PipelineArtifactCaching.SaveCacheV1.SaveCache@1	
```

Comment:
Set `files.trimTrailingWhitespace` to `true` 👍

Reply:
If only I used VS Code (I used the GH in browser editor 🤦‍♂️)